hivemind/moe/client/expert.py

```diff
@@ -57,12 +50,12 @@ def uid(self):
         return self._info.uid
 
     @property
-    def server_peer_info(self):
-        return self._info.peer_info
+    def server_peer_id(self) -> PeerID:
```

Comment:
```suggestion
    def peer_id(self) -> PeerID:
```

I'd suggest to rename it like this because (a) it's clear that `RemoteExpert.peer_id` is the server's peer ID (not our own) and (b) many `hivemind` classes have the `peer_id` property and it would be cool to be in line with them.

@GreenFatGuy Are you okay with this change? (@justheuristic asked for your confirmation)